docs/sql-ref-ansi-compliance.md

```diff
@@ -257,6 +257,15 @@ The behavior of some SQL operators can be different under ANSI mode (`spark.sql.
   - `map_col[key]`: This operator throws `NoSuchElementException` if key does not exist in map.
   - `GROUP BY`: aliases in a select list can not be used in GROUP BY clauses. Each column referenced in a GROUP BY clause shall unambiguously reference a column of the table resulting from the FROM clause.
 
+### Special functions for using the ANSI dialect
+
+After turning ANSI mode on, if you expect some of your SQL operations to not throw exceptions on errors, as Spark's default behavior, you can use the following functions.
```

Comment:
How about this?

```suggestion
When ANSI mode is on, it throws exceptions for invalid operations. You can use the following functions for SQL operations to suppress such exceptions as if ANSI mode is disabled.
```

Reply:
It's not "as if ANSI mode is disabled":
1. The acceptable input of TRY_CAST is the same as "CAST" in ANSI mode
2. When ANSI mode is disabled, `1+Int.MaxValue` is -2147483648, while `TRY_ADD(1, Int.MaxValue)` is null